lib/tlsconfig.go

```diff
@@ -61,14 +61,18 @@ var SupportedTLSCipherSuites = map[string]uint16{
 	"TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256": tls.TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
 	"TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384":   tls.TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,
 	"TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384": tls.TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,
+	"TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256": tls.TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
+	"TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305":    tls.TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,
+	"TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305":  tls.TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,
```

Comment:
Compare this list (which we probably should have found a way to auto-generate a long time ago :sweat_smile: ) with the [constaints in the Go source](https://golang.org/pkg/crypto/tls/#pkg-constants) and we seem to be missing the `TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256` algorithm, as well as the new TLS 1.3 ones: 
```
// TLS 1.3 cipher suites.
TLS_AES_128_GCM_SHA256
TLS_AES_256_GCM_SHA384
TLS_CHACHA20_POLY1305_SHA256
```